packages/modal/storybook/Modal.js

```diff
@@ -59,21 +59,35 @@ storiesOf('Modal', module)
       width={['80vw', '400px', '500px']}
     />
   ))
-  .add('With ModalHeader (and scrollLock!)', () => (
-    <ModalStory
-      header={
-        <ModalHeader
-          title="Modal title"
-          onClose={() => {
-            alert('This shuold handle close')
-          }}
+  .add('With ModalHeader (and ScrollLock!)', () => {
+    // Generate content to demonstrate a scrollable <body>
+    const contentLines = []
+    for (let i = 0; i < 100; i++) {
+      contentLines.push(i)
+    }
```

Comment:
If you need an array with 100 slots to iterate but `.map` at L72, seems that a new array with 100 empty elements, without the `for` loop to fill in the integers for all elements in array.

```js
const contentLines = new Array(100)
```

Reply:
I actually tried that at first, but mapping over it didn't do anything. This works, though.

```js
[...Array(100).keys()]
```